protostar/profiler/contract_profiler.py

```diff
@@ -0,0 +1,319 @@
+# pylint: disable=invalid-name
```

Comment:
Consider creating `README.md` in the profiler module with high level overview.

Reply:
#983